src/orm/base_model/index.ts

```diff
@@ -1802,6 +1802,22 @@ class BaseModelImpl implements LucidRow {
       .processAllForOne(this, queryClient)
   }
 
+  /**
+   * Load relationships onto the instance, but only if they are not
+   * already preloaded
+   */
+  async loadOnce(relationName: any) {
+    this.ensureIsntDeleted()
+
+    if (!this.$isPersisted) {
+      throw new Exception('Cannot lazy load relationship for an unpersisted model instance')
+    }
+
```

Comment:
Wouldn't this be handled by the `load` method itself?

Reply:
Correct!